src/lib.rs

```diff
@@ -266,11 +325,14 @@ fn eth_sent_store(events: contract::Events, o: StoreAddBigInt) {
 
 fn graph_activepool_out(events: &contract::Events, tables: &mut EntityChangesTables, clock: Clock, eth_sent_store: StoreGetBigInt) {
     // Loop over all the abis events to create table changes
+    let contract_name = get_name();
```

Comment:
This is the function that does the RPC call. Where could this be moved to optimise the substream, and make this module run faster?